src/Components/IndexScene/IndexScene.tsx

```diff
@@ -47,16 +49,14 @@ export interface IndexSceneState extends SceneObjectState {
   // contentScene is the scene that is displayed in the main body of the index scene - it can be either the service selection or service scene
   contentScene?: SceneObject;
```

Comment:
Now that we're refactoring this, I find this concept of `body` and `contentScene` confusing. Body seems to be fixed to always be a `LayoutScene`, so maybe we should remove it? Or at least create the instance in the constructor?

https://github.com/grafana/explore-logs/blob/3da4446d8b66abc58e187826f587d8e4130e5500/src/Components/IndexScene/IndexScene.tsx#L91

Or maybe the `body` should either be ServiceSelectionScene or ServiceScene. I think it would simplify this component a bit more.

Reply:
Yeah I'll take a look, the constructor isn't between routes if the scene is cached, but the activation function is, so as long as there's nothing in the layoutScene that needs to be re-inited we should be fine to move this to the constructor

Reply:
Fixed.